libbeat/docs/shared/obs-apps.asciidoc

```diff
@@ -0,0 +1,56 @@
+. Ingest data from other sources by installing and configuring other Elastic
+{beats}:
++
+--
+[options="header"]
+|===
+|Elastic {beats} | To capture
+
+ifeval::["{beatname_lc}"!="metricbeat"]
+|{metricbeat-ref}/metricbeat-installation-configuration.html[{metricbeat}]
+|Infrastructure metrics
+endif::[]
+ifeval::["{beatname_lc}"!="filebeat"]
+|{filebeat-ref}/filebeat-installation-configuration.html[{filebeat}]
+|Logs
+endif::[]
+ifeval::["{beatname_lc}"!="winlogbeat"]
+|{winlogbeat-ref}/winlogbeat-installation-configuration.html[{winlogbeat}] 
+|Windows event logs
+endif::[]
+ifeval::["{beatname_lc}"!="heartbeat"]
+|{heartbeat-ref}/heartbeat-installation-configuration.html[{heartbeat}]
+|Uptime information
+endif::[]
+|{apm-get-started-ref}/index.html[APM]
```

Comment:
@bmorelli25 This reference to APM is breaking because it looks like APM docs aren't being built in the 7.x branch. Should I use something different? Remove APM for now?

Reply:
@dedemorton, we need to add something like this to `versions.asciidoc`:

```
///////
APM does not build n.x documentation. Links from .x branches should point to master instead
///////
ifeval::["{source_branch}"=="7.x"]
:apm-server-ref:       {apm-server-ref-m}
This conversation was marked as resolved by bmorelli25
:apm-server-ref-v:     {apm-server-ref-m}
:apm-overview-ref-v:   {apm-overview-ref-m}
endif::[]
```

See this related PR in the elasticsearch repo for more information. I can add this to the Beats repo if you'd like.

Reply:
Awesome. Thanks! I'll just add it as part of this PR.